lib/node/http.js

```diff
@@ -498,6 +498,13 @@ class HTTP extends Server {
       return null;
     });
 
+    socket.hook('get block peer', (...args) => {
```

Comment:
do we need this?

Reply:
Yes, it is being called by lib/client/node